frontend/src/components/Footer/Footer.tsx

```diff
@@ -11,25 +7,12 @@ interface FooterProps {
 }
 
 const Footer = async ({ isSmall }: FooterProps) => {
-  const { t } = await useTranslation('common')
-  const isRunningInApp = headers().get('referer')?.includes('android-app://fit.crab')
-
-  return isRunningInApp
-    ? null // Cannot show external donation link in an Android app
-    : <footer
-      id="donate" // Required to allow scrolling directly to the footer
-      className={makeClass(styles.footer, isSmall && styles.small)}
-    >
-      <span>{t('donate.info')}</span>
-      <Button
-        isSmall
-        title={t('donate.title')}
-        href="https://ko-fi.com/A06841WZ"
-        target="_blank"
-        rel="noreferrer noopener payment"
-        style={{ whiteSpace: 'nowrap' }}
-      >{t('donate.button')}</Button>
-    </footer>
-}
+  return <footer className={makeClass(styles.footer, isSmall && styles.small)}>
+    <p>
+      Based on <a href="https://github.com/GRA0007/crab.fit">Crab Fit</a> from <a href="https://bengrant.dev/"> Ben Grant</a> for custom style and content of pages. 
```

Comment:
```suggestion
      Based on <a href="https://github.com/GRA0007/crab.fit">Crab Fit</a> from <a href="https://bengrant.dev/"> Ben Grant</a>. 
```

I find the formulation unclear. The original repo contains the logic, and the fork was created to change the UI.